docs/tools_user_guide.md

```diff
@@ -277,15 +279,17 @@ Each device type has its own default runtime, as indicated below. Valid values i
 - `ort`: ONNX Runtime (default for `x86` device type).
 - `trt`: Nvidia TensorRT (default for `nvidia` device type).
 - `groq`: GroqFlow (default for `groq` device type).
-- [future] `pytorch1`: PyTorch 1.x-style eager execution.
-- [future] `pytorch2`: PyTorch 2.x-style compiled graph execution.
+- `torch`: PyTorch default execution.
+- `torch_compiled`: PyTorch 2.x-style compiled graph execution.
 - [future] `ort-*`: Specific [ONNX Runtime execution providers]
```

Comment:
Consistency: `ort-*` vs. `torch_*`, suggest using a `-` or `_` in both naming schemes.

Reply:
Solved! `torch_compiled` is now `torch-compiled`